### PR TITLE
Fix Golint complains (#6)

### PR DESCRIPTION
**Pull request by metarsit**
#6 

I am just fixing whatever `golint` complains on the .go files

However, there are 2 things I am unsure about.
The description for `Read()` and `Take()`might be inaccurate just because the source code does not tell me enough what it does exactly, so It is my guessing work there. Please check and let me know @kyoungho

### CODE REVIEW

File: rticonnextdds-connector.go [hunk 1]
```diff
@@ -372,69 +387,77 @@ func (instance *Instance) SetRune(fieldName string, value rune) error {
...
-	err = instance.SetJson(jsonData)
+	err = instance.SetJSON(jsonData)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
+// Read is a function to initialize to read the DDS messages
```
Comment (kyoungho) [inline rticonnextdds-connector.go]: Read is a function to read DDS samples from the DDS DataReader and allow access them via the Connector Samples. The Read function does not remove DDS samples from the DDS DataReader's receive queue.

File: rticonnextdds-connector.go [hunk 2]
```diff
@@ -372,69 +387,77 @@ func (instance *Instance) SetRune(fieldName string, value rune) error {
...
 		return err
 	}
 
-	// The C function does not return errors. In the futurue, we will update this when supported in the C layer
+	// The C function does not return errors. In the future, we will update this when supported in the C layer
 	C.RTIDDSConnector_read(unsafe.Pointer(input.connector.native), input.nameCStr)
 	return nil
 }
 
+// Take is a function to initialize to read and take the DDS messages
```
Comment (kyoungho) [inline rticonnextdds-connector.go]: Take is a function to take DDS samples from the DDS DataReader and allow access them via the Connector Samples. The Take function removes DDS samples from the DDS DataReader's receive queue.

**Comment by metarsit** [comment]
Got it. I will fix the 2 comments and this PR shud be gd to go if u r :)

**Comment by metarsit** [comment]
@kyoungho I have updated! :)

**Comment by kyoungho** [comment]
Thanks! I will merge your request after you are done with CLA. Let me know!

**Comment by metarsit** [comment]
@kyoungho i need to talk to my company before I sign that CLA.. i will update u closely..

2nd thought: should we put gofmt golint into travis? He seems like a nice guy.. hahahaha

**Comment by kyoungho** [comment]
@metarsit OK. Let me know if you have any update. That will be nice if we can add golint and gofmt to Travis. Agree. He is a really nice guy :)

**Comment by metarsit** [comment]
@kyoungho I have signed the CLA. :) Also, if this PR is merged, can you help to update the godoc? :)

**Comment by kyoungho** [comment]
Awesome! I will merge the PR and update the godoc.

**Pull request merged**